### ISSUE TITLE
what is the meaning of the identity loss and training result?

### ISSUE THREAD

**Issue by Adherersun** [junyanz/pytorch-CycleGAN-and-pix2pix#373]
1.The first question
In the paper, it says：

> In particular, we adopt the technique of Taigman et al. [45] and regularize the generator to be near an identity mapping when real samples of the target domain are provided as the input to the generator:
![image](https://user-images.githubusercontent.com/24805080/45282615-fe45f500-b50d-11e8-89f5-b05785314636.png)

What is the meaning of the identity loss?
Can anyone tell the meaning of this formula and explain this formula specifically? thank you very much

This is the result of epoch [200]:
![image](https://user-images.githubusercontent.com/24805080/45282736-60065f00-b50e-11e8-863a-1da6878b6643.png)

What is the meaning of the rec_A and idt_A?Is picture A randomly extracted from the training data set?

**Comment by junyanz** [comment]
See these two posts [1](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/issues/322) and [2](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/pull/362).
